ckks/ckks_test.go

```diff
@@ -713,13 +713,32 @@ func testEvaluatorMulAndAdd(tc *testContext, t *testing.T) {
 
 		tc.evaluator.MulRelinAndAdd(ciphertext2, ciphertext2, ciphertext1)
 
-		//fmt.Println(ciphertext1.Value[2].Coeffs[0][:4])
-
 		require.Equal(t, ciphertext1.Degree(), 1)
 
 		verifyTestVectors(tc.params, tc.encoder, tc.decryptor, values1, ciphertext1, tc.params.LogSlots(), 0, t)
 	})
 
+	t.Run(GetTestName(tc.params, "Evaluator/MulAndAdd/ct0*ct1->ct2"), func(t *testing.T) {
```

Comment:
Add the same test for MulRelinAndAdd ?

Reply:
It is not necessary, the relinearization is a second step, independent from what is happening internally before it, so it only needs to be tested once as it is not affected by the inputs.